quiz.js

```diff
@@ -137,11 +139,6 @@ function showCorrectAnswer() {
   showAnswerButton.style.display = 'none';
 }
 
-function setAnswerHint(answerText) {
-  let answerParagraph = document.querySelectorAll('main p')[1];
```

Comment:
to dobry pomysł, żeby po prostu kolorować prawidłową odpowiedź. Wtedy nie jest na ekranie dwa razy